### PR TITLE
chore: add library_type to .repo-metadata.json for handwritten libraries

### PR DESCRIPTION
**Pull request by trambui09**
For handwritten libraries in this repo, add library_type field to the .repo-metadata.json file and set as GAPIC_MANUAL. This PR is part of internal issue #184315365.

### CODE REVIEW

File: google-cloud-asset-v1beta1/.repo-metadata.json [hunk 1]
```diff
@@ -7,5 +7,6 @@
     "name_pretty": "Cloud Asset V1beta1 API",
     "product_documentation": "https://cloud.google.com/asset-inventory/",
     "repo": "googleapis/google-cloud-ruby",
-    "requires_billing": true
+    "requires_billing": true,
+    "library_type": "GAPIC_MANUAL"
```
Comment (dazuma) [inline google-cloud-asset-v1beta1/.repo-metadata.json:11]: Sorry, we can drop this one. This library is actually dead, and I've been meaning to delete it.

File: google-cloud-compute-v1/.repo-metadata.json [hunk 2]
```diff
@@ -5,5 +5,6 @@
     "name_pretty": "Google Cloud Compute V1 API",
     "release_level": "alpha",
     "repo": "googleapis/google-cloud-ruby",
-    "requires_billing": true
+    "requires_billing": true,
+    "library_type": "GAPIC_MANUAL"
```
Comment (dazuma) [inline google-cloud-compute-v1/.repo-metadata.json:9]: Hold off on this one for now. It's actually generated; Slava is still working on getting it into synth/owlbot.

File: google-cloud-core/.repo-metadata.json [hunk 3]
```diff
@@ -2,5 +2,6 @@
     "name": "core",
     "language": "ruby",
     "distribution_name": "google-cloud-core",
-    "client_documentation": "https://googleapis.dev/ruby/google-cloud-core/latest"
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-core/latest",
+    "library_type": "GAPIC_MANUAL"
```
Comment (dazuma) [inline google-cloud-core/.repo-metadata.json:6]: This one is CORE.

File: google-cloud-errors/.repo-metadata.json [hunk 4]
```diff
@@ -2,5 +2,6 @@
     "name": "errors",
     "language": "ruby",
     "distribution_name": "google-cloud-errors",
-    "client_documentation": "https://googleapis.dev/ruby/google-cloud-errors/latest"
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-errors/latest",
+    "library_type": "GAPIC_MANUAL"
```
Comment (dazuma) [inline google-cloud-errors/.repo-metadata.json:6]: This one is CORE

File: google-cloud-debugger/.repo-metadata.json [hunk 5]
```diff
@@ -2,5 +2,6 @@
     "name": "clouddebugger",
     "language": "ruby",
     "distribution_name": "google-cloud-debugger",
-    "client_documentation": "https://googleapis.dev/ruby/google-cloud-debugger/latest"
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-debugger/latest",
+    "library_type": "GAPIC_MANUAL"
```
Comment (dazuma) [inline google-cloud-debugger/.repo-metadata.json:6]: Now that I look at these, I think this one is a weird case, and probably better labeled as INTEGRATION.

File: google-cloud-error_reporting/.repo-metadata.json [hunk 6]
```diff
@@ -2,5 +2,6 @@
     "name": "clouderrorreporting",
     "language": "ruby",
     "distribution_name": "google-cloud-error_reporting",
-    "client_documentation": "https://googleapis.dev/ruby/google-cloud-error_reporting/latest"
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-error_reporting/latest",
+    "library_type": "GAPIC_MANUAL"
```
Comment (dazuma) [inline google-cloud-error_reporting/.repo-metadata.json:6]: This one is probably also INTEGRATION.

File: google-cloud-trace/.repo-metadata.json [hunk 7]
```diff
@@ -2,5 +2,6 @@
     "name": "cloudtrace",
     "language": "ruby",
     "distribution_name": "google-cloud-trace",
-    "client_documentation": "https://googleapis.dev/ruby/google-cloud-trace/latest"
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-trace/latest",
+    "library_type": "GAPIC_MANUAL"
```
Comment (dazuma) [inline google-cloud-trace/.repo-metadata.json:6]: And this one also INTEGRATION.

**Review by dazuma — CHANGES_REQUESTED** [review]

**Comment by trambui09** [comment]
Just made the proper edits for the library_type

**Review by dazuma — APPROVED** [review]

**Pull request merged**